web/viewer.html

```diff
@@ -686,6 +686,18 @@
 <!--#endif-->
       </div>  <!-- dialogContainer -->
 
+      <div id="toastContainer">
```

Comment:
Here, and elsewhere in the patch, please update this name as well.

Reply:
I was actually curious if we should remove the div entirely since it seems that we're not going to focus much on generalizing for future popups like this, what is your preferred name for this div? `editorUndoBarContainer` seems to be a mouthful but I'm more than happy to accept any suggestions.

Reply:
That seems like a good idea to me. (We can always extend the functionality in the future, if actually needed, but let's keep things as simple as possible for now.)